packages/components/src/internal/components/search/utils.ts

```diff
@@ -179,7 +179,16 @@ export function getSampleFinderCommonConfigs(
                 baseFilters.push(filter);
             }
         } else {
-            baseFilters.push(Filter.create(cardColumnName + '/Name', null, Filter.Types.NONBLANK));
+            const pkColName = 'Name';
+            const filter = getExpDescendantOfFilter(schemaQuery, [{
```

Comment:
Curious to know why the previous query was wrong.

Reply:
Also, does this mean existing saved view with the old filter is broken?

Reply:
I think we originally added this during a very early phase of the sample finder work, but it wasn't actually possible to create this type of filter directly from the Sample Finder UI (as far as I could tell), so the code became unused. With this code path, you can get this filter again but I wasn't seeing the correct results. When I compared the network tab filtering for the previous code and what I got when I manually added a "Name is not null" filter, it was different so I changed it to match that by using the `getExpDescendantOfFilter` helper. Let me know if you see any issues in manual testing or unexpected results in the grids.